yo/db.py

```diff
@@ -90,47 +88,47 @@ class Priority(enum.Enum):
 notifications_table = sa.Table(
     'yo_notifications',
     metadata,
-    sa.Column(
```

Comment:
Why use a longer value? Why not define it at in the table definition to avoid having to set it anywhere else?

Reply:
Can you clarify what you mean by this?

If you mean the primary key (nid), i've set it to the length of a UUID